docs/start-building/index.txt

```diff
@@ -42,8 +42,26 @@ Pick your library, and start building!
 
 If you would like to see something added to this page, please `get in touch`_.
 
+.. TIP::
+
+    CrateDB supports the `PostgreSQL wire protocol`_. Accordingly, many clients
+    that work with PostgreSQL also work with CrateDB.
+
+    You can try this out for yourself by configuring a normal PostgreSQL
+    connection. All you need to do is point the connection to a CrateDB server
+    instead of a PostgreSQL server. The rest of the database connection setup
```

Comment:
Do you think it is worth also referring to the [User management](https://crate.io/docs/crate/reference/en/latest/admin/user-management.html) or [Authentication](https://crate.io/docs/crate/reference/en/latest/admin/auth/index.html) - e.g. that by default a `crate` user exists that can be used with an empty password (if the client connects from the same host)?

Not sure if it is already too much detail. But the "okay, and what are the credentials?" might be a question a user trying out a PostgreSQL client has.

Reply:
thanks! I made some changes!